### PR TITLE
Convert swift4

### PR DESCRIPTION
**Pull request by JamesDurham**
A version of Realm Model Generator in swift 4.

### CODE REVIEW

File: RealmModelGenerator/AttributeDetailView.swift [hunk 1]
```diff
@@ -45,28 +46,28 @@ class AttributeDetailView: NibDesignableView, NSTextFieldDelegate {
...
-        get { return self.ignoredCheckBox.state == 1 ? true : false }
+        set { self.ignoredCheckBox.state = newValue ? NSControl.StateValue.on : NSControl.StateValue.off }
+        get { return self.ignoredCheckBox.state == NSControl.StateValue.on ? true : false }
     }
     
     @IBInspectable var isIndexed:Bool {
-        set { self.indexedCheckBox.state = newValue ? 1 : 0 }
-        get { return self.indexedCheckBox.state == 1 ? true : false }
+        set { self.indexedCheckBox.state = newValue ? NSControl.StateValue.on : NSControl.StateValue.off }
+        get { return self.indexedCheckBox.state == NSControl.StateValue.on ? true : false }
```
Comment (berbschloe) [inline RealmModelGenerator/AttributeDetailView.swift]: use shorthand `.on` and `.off` where applicable.

File: RealmModelGenerator/MainVCExtension.swift [hunk 2]
```diff
@@ -8,7 +8,11 @@
 
 import Cocoa
 
+// --MARK-- temporary workaround
+fileprivate let VERSIONVC: NSStoryboard.SceneIdentifier = NSStoryboard.SceneIdentifier(rawValue: "VersionVC")
```
Comment (berbschloe) [inline RealmModelGenerator/MainVCExtension.swift]: static `let` are valid inside of extensions.
You could also declare at usage site. keeps the code "localized"

File: RealmModelGenerator/NibDesignable.swift [hunk 3]
```diff
@@ -72,7 +72,7 @@ extension NSView {
         return String(type(of: self).description().split(separator: ".").last!)
     }
     
-    public func nibDidLoad() {}
+    // public func nibDidLoad() {}
```
Comment (berbschloe) [inline RealmModelGenerator/NibDesignable.swift]: remove commented out method. It's added below

**Review by berbschloe — CHANGES_REQUESTED** [review]
Just a couple nits. Other than that. Looks really good.

File: RealmModelGenerator/AttributeDetailView.swift (hunk 1 above)
Comment (JamesDurham) [reply RealmModelGenerator/AttributeDetailView.swift]: Done

File: RealmModelGenerator/MainVCExtension.swift (hunk 2 above)
Comment (JamesDurham) [reply RealmModelGenerator/MainVCExtension.swift]: SceneIdentifier should be in seprate file (and other ids) 
just not now

File: RealmModelGenerator/NibDesignable.swift (hunk 3 above)
Comment (JamesDurham) [reply RealmModelGenerator/NibDesignable.swift]: Done

**Review by berbschloe — APPROVED** [review]

**Pull request merged**